rednotebook/util/utils.py

```diff
@@ -202,3 +203,14 @@ def flush(self):
     def close(self):
         for stream in self.streams:
             stream.close()
+
+def safe_strxfrm(value):
+    """
+    Safely apply locale-aware sorting. If locale.strxfrm fails, fall back to default sorting.
+    """
+    try:
+        return locale.strxfrm(value)
+    except OSError:
+        return value
+    except:
```

Comment:
Do we really need the bare except? I think it's better to remove it since we have the except for OSError.